eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Utxo.hs

```diff
@@ -65,5 +251,273 @@ allegraToConwayUtxoPredFailure = \case
   Allegra.OutputTooSmallUTxO x -> OutputTooSmallUTxO x
   Allegra.UpdateFailure x -> absurdEraRule @"PPUP" @era x
   Allegra.OutputBootAddrAttrsTooBig xs -> OutputBootAddrAttrsTooBig xs
-  Allegra.TriesToForgeADA -> TriesToForgeADA
+  Allegra.TriesToForgeADA -> error "How should I deal with this?"
```

Comment:
Is this okay or should I try to come up with something along the lines of `absurdEraRule?`